catroid/src/main/java/org/catrobat/catroid/content/bricks/StopScriptBrick.java

```diff
@@ -110,6 +107,7 @@ public View getPrototypeView(Context context) {
 	}
 
 	private ArrayAdapter<String> createArrayAdapter(Context context) {
+		String[] spinnerValue = new String[3];
 		spinnerValue[BrickValues.STOP_THIS_SCRIPT] = context.getString(R.string.brick_stop_this_script);
 		spinnerValue[BrickValues.STOP_ALL_SCRIPTS] = context.getString(R.string.brick_stop_all_scripts);
 		spinnerValue[BrickValues.STOP_OTHER_SCRIPTS] = context.getString(R.string.brick_stop_other_scripts);
```

Comment:
As you can see, it doesn't make sense to serialize __localized__ spinner values.